.github/workflows/odh-notebooks-sync.yml

```diff
@@ -0,0 +1,80 @@
+# The aim of this GitHub workflow is to update the pyproject.toml to sync with ODH Jupyter Trash Cleanup release.
+name: Sync ODH-notebooks with ODH Jupyter Trash Cleanup release
+on:
+  workflow_dispatch:
+    inputs:
+      upstream-repository-organization:
+        required: true
+        description: "Owner of target upstream notebooks repository used to open a PR against"
+        default: "opendatahub-io"
+      notebooks-target-branch:
+        required: true
+        description: "Target branch of upstream repository"
+        default: "main"
+      repository-organization:
+        required: true
+        description: "Owner of origin notebooks repository used to open a PR"
+        default: "odh-ide-extensions"
+      repository-name:
+        required: true
+        description: "Name of the target repository"
+        default: "notebooks"
+      odh_jupyter_trash_cleanup_release_version:
+        required: true
+        description: "Provide version of the ODH Jupyter Trash Cleanup release"
+
+env:
+  BRANCH_NAME: ${{ github.event.inputs.notebooks-target-branch }}
+  ODH_JUPYTER_TRASH_CLEANUP_RELEASE_VERSION: ${{ github.event.inputs.odh_jupyter_trash_cleanup_release_version }}
+  UPDATER_BRANCH: odh-sync-updater-${{ github.run_id }}
+  UPSTREAM_OWNER: ${{ github.event.inputs.upstream-repository-organization }}
+  REPO_OWNER: ${{ github.event.inputs.repository-organization }}
+  REPO_NAME: ${{ github.event.inputs.repository-name }}
+  GITHUB_TOKEN: ${{ secrets.ODH_JUPYTER_TRASH_CLEANUP_MACHINE_ACCOUNT_TOKEN }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repository and Sync
+        run: |
+          git clone https://x-access-token:${GITHUB_TOKEN}@github.com/$REPO_OWNER/$REPO_NAME.git $REPO_NAME
```

Comment:
I believe that we agreed not to have our own fork of the notebooks repository but we'll be doing the PR directly into their repo. Let's confirm this with others.

Reply:
We can utilize https://github.com/ide-developer/notebooks,
and the bot ide-developer, for this development.